app/services/sources/sources-data.ts

```diff
@@ -0,0 +1,120 @@
+    supportList: imageSupport
+  },
+  text_gdiplus: {
+    name: $t('Text (GDI+)'),
+    description: $t('Add text to your scene and adjust its style.'),
+    demoFilename: 'text.png',
+    supportList: [...colorSupport, $t('System Fonts'), $t('System Sizes')]
+  },
+  monitor_capture: {
+    name: $t('Display Capture'),
+    description: $t('Capture your entire computer monitor.'),
+    demoFilename: 'display-capture.png',
+    supportList: [$t('Primary monitor'), $t('Secondary monitor')]
+  },
+  window_capture: {
+    name: $t('Window Capture'),
+    description: $t('Capture a specific window that\'s open on your computer.'),
+    demoFilename: 'window-capture.png',
+    supportList: [$t('Compatible with most modern browsers and programs')]
+  },
+  game_capture: {
+    name: $t('Game Capture'),
+    description: $t('Capture a game you\'re playing on your computer.'),
+    demoFilename: 'game-capture.png',
+    supportList: [$t('Built in works with most modern computer games')]
+  },
+  dshow_input: {
+    name: $t('Video Capture Device'),
+    description: $t('Select from your build in USB webcam or an external.'),
+    demoFilename: 'video-capture.png',
+    supportList: [
+      $t('Built in webcam'),
+      $t('Logitech webcam'),
+      $t('Capture cards (Elgato, Avermedia, BlackMagic)')
+    ]
+  },
+  ndi_input: {
+    name: $t('NDI source'),
+    description: $t('Allow you to capture NDI output streams.')
+  },
+  'decklink-input': {
+    name: $t('Blackmagic Device'),
+    description: $t('Capture the feed your decklink device is capturing.'),
+    demoFilename: 'sources.png',
+    supportList: [$t('Works with most of the recent Blackmagic cards.')]
+  },
+  open_vr_capture: {
+    name: $t('OpenVR Capture'),
+    description: $t('Directly capture the OpenVR monitoring video buffer of your HMD.'),
+    demoFilename: 'vr-capture.png',
+    supportList: ['OpenVR', 'SteamVR']
+  },
+  liv_capture: {
+    name: $t('LIV Client Capture'),
+    description: $t(
+      'Directly capture the LIV compositor output, reducing load and simplifying setup for Mixed Reality.'
+    ),
+    demoFilename: 'vr-capture.png',
+    supportList: ['LIV']
+  },
+  wasapi_input_capture: {
+    name: $t('Audio Input Capture'),
+    description: $t(
+      'Any device that attaches to a computer for the purpose of capturing sound, such as music or speech.'
+    ),
+    demoFilename: 'audio-input.png',
+    supportList: [
+      $t('Built in microphones'),
+      $t('USB microphones'),
+      $t('Other USB devices')
+    ]
+  },
+  wasapi_output_capture: {
+    name: $t('Audio Output Capture'),
+    description: $t('Captures your desktop audio for the purpose of playing sound, such as music or speech.'),
+    demoFilename: 'audio-output.png',
+    supportList: [$t('Desktop audio')]
+  },
+  // vvv Not sure if this is still supported, there is no showcase data for it vvv
+  text_ft2_source: {
+    name: $t('Text (FreeType 2)'),
+    description: ''
+  }
+});
```

Comment:
I feel like this object is what `SourcesService.getAvailableSourcesTypesList` have to return

Reply:
possibly! i just didn't want to change too much (at least in the first version of this) in case that breaks something elsewhere